redwood-runtime/src/commonMain/kotlin/app/cash/redwood/LayoutModifier.kt

```diff
@@ -94,6 +96,7 @@ public interface LayoutModifier {
    */
   // The companion object implements `LayoutModifier` so that it may be used as the start of a
   // modifier extension factory expression.
+  @ObjCName("Companion")
   public companion object : LayoutModifier {
```

Comment:
Named this as `LayoutModifier.Companion` for consistency, but might be better as `LayoutModifier.empty` or something else?

Reply:
Renamed to `EmptyLayoutModifier`.